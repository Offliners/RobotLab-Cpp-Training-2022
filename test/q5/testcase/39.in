(3-19/20)+20/14-19+19*11+(10-(19+17)+8-2+14)-7=+(15-5+10-13-18-(5-17))=-(9/8-7+7*2-2/17/6/1-2+13*9+16*5)%14=-8/11+8=
