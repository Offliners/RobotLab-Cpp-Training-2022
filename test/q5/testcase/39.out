22
-1
26
31
62
123
